File: engine/engine-model/src/main/scala/tools/packet/statusinfo/CompleteQuest.scala
package tools.packet.statusinfo

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class CompleteQuest(private var _questId: Short, private var _time: Long) extends PacketInput {
  def questId: Short = _questId

  def time: Long = _time

  override def opcode(): SendOpcode = SendOpcode.SHOW_STATUS_INFO
}

File: engine/engine-model/src/main/java/tools/packet/statusinfo/CompleteQuest.java
package tools.packet.statusinfo;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record CompleteQuest(Short questId, Long time) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SHOW_STATUS_INFO;
   }
}